front-end/src/screens/CreatePost/index.tsx

```diff
@@ -108,6 +111,12 @@ const CreatePost = ({ className }:Props): JSX.Element => {
 						rules={{ required: true }}
 					/>
 
+					<Form.Group className="hidden">
+						<Form.Field>
+							<input type='checkbox' onChange={onPollChanged} /> <span className='text-muted'>Add a poll to this discussion</span>
```

Comment:
Please use the same toggle that we have for notification.

Reply:
done